lib/chef/knife/maas/version.rb

```diff
@@ -1,7 +1,7 @@
 class Chef
   class Knife
     module Maas
-      VERSION = "1.2.0"
+      VERSION = '1.2.0'
```

Comment:
Yep, it's still a WIP but I'll certainly bump the version.  I haven't actually tested any of this code because I don't have a mass install yet.  I also want to refactor those sleeps before this gets in.